can/generated/uwrt_mars_rover_can_enums.h

```diff
@@ -0,0 +1,162 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY
+
+#pragma once
+
+namespace HWBRIDGE {
+
+constexpr uint32_t ROVER_CANBUS_FREQUENCY_HZ = 1000000;
+
+typedef double CANSignalValue_t;
+
+typedef struct {
+  uint8_t raw[8];
+} CANMsgData_t;
+
+enum CANBUSID {
```

Comment:
prefer enum classes over enums where appropriate

Reply:
I think this is autogenerated? So changing this would mean needing to fork the upstream repo for the generator tool and making changes there

Reply:
this one's custom auto-generated, I can change it in the python script to make them enum classes